### PR TITLE
fix(reset): remove typography rules from reset - INNO-949

### PR DESCRIPTION
**Pull request by yhuard**
It's the final PR regarding the reset 🕐🎉🍾

Two options:
1. apply a class on the most top-level acceptable DOM element (easy way: body), like I did here with `.ecl-typography`
2. refactor almost all the components to explicitly apply the font rules, _à la_ https://github.com/carbon-design-system/carbon-components/search?l=SCSS&q=font-family&type=

I went with option 1 because it was faster but option 2 is more robust. It could be part of the normalization process :D cc @emeryro

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 9b66ae0a7d50a2b4b62451a9f13b43d71afba4f1

https://deploy-preview-629--europa-component-library.netlify.com

**Comment by yhuard** [comment]
TBD: merging `typography-heading` and `typography-paragraph` into typography` while moving `typography-list` into a component

File: src/systems/ec/ec-style/ec-style-typography/ec-style-typography/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Typography
+
+Be careful when applying the `.ecl-typography` class.
```
Comment (kalinchernev) [inline src/systems/ec/ec-style/ec-style-typography/ec-style-typography/README.md]: Could you please elaborate?

**Review by kalinchernev — COMMENTED** [review]
The approach seems ok to me. Applying a class selectively on particular element to scope the styles looks better than applying the rules on individual components. It'll also be easier to maintain the system in this way.

Only 2 unclear items:
- is it worth keeping empty twig templates in generic components
- readme file warns about usage but doesn't give details

**Review by kalinchernev — APPROVED** [review]
Thanks!

**Pull request merged**